tests/test_inner.py

```diff
@@ -129,6 +133,14 @@ class PreRaises(RaisesMixIn, PreBuildPlugin):
     key = 'pre_raises'
 
 
+class BuildStepRaises(RaisesMixIn, BuildStepPlugin):
+    """
+    This plugin must run and cause the build to abort.
+    """
+
+    key = 'buildstep_raises'
```

Comment:
Let's be consistent with what we call these objects. The 'watched' one is called BuildWatched with key 'build_watched'. Let's choose either "Build..."+"build_..." or "BuildStep..."+"buildstep_...".